# HG changeset patch
# User Rik <rik@octave.org>
# Date 1437507725 25200
#      Tue Jul 21 12:42:05 2015 -0700
# Node ID c5343519df789cbd8a9ba5f05325874e17111b93
# Parent  07ddaaeb90407fa32f82413a7b23b270c63e1044
build: Add missing scripts from image/private/ dir to tarball.

* scripts/image/module.mk: Add colorspace_conversion_input_check.m and
colorspace_conversion_revert.m to scrips_image_PRIVATE_FCN_FILES var.

diff --git a/scripts/image/module.mk b/scripts/image/module.mk
--- a/scripts/image/module.mk
+++ b/scripts/image/module.mk
@@ -1,16 +1,18 @@
 FCN_FILE_DIRS += \
   scripts/image \
   scripts/image/private
 
 scripts_image_PRIVATE_FCN_FILES = \
   scripts/image/private/__imfinfo__.m \
   scripts/image/private/__imread__.m \
   scripts/image/private/__imwrite__.m \
+  scripts/image/private/colorspace_conversion_input_check.m \
+  scripts/image/private/colorspace_conversion_revert.m \
   scripts/image/private/imageIO.m \
   scripts/image/private/imwrite_filename.m \
   scripts/image/private/ind2x.m
 
 scripts_image_FCN_FILES = \
   scripts/image/autumn.m \
   scripts/image/bone.m \
   scripts/image/brighten.m \
